building.rb

```diff
@@ -1,2 +1,29 @@
 class Building
+	attr_accessor :address, :apartments
+
+	def initialize(args={})
+
+		@address = args[:address]
+		@apartments = args[:apartments]
+	end
+
+	def total_room_count
+		@apartments.map {|apartment| apartment.room_count}.reduce(:+)
+	end
+
+	def total_monthly_revenue
+		@apartments.map {|apartment| apartment.monthly_rent}.reduce(:+)
+	end
+
+	def apartments_by_rent
+		@apartments.sort_by {|apartment| apartment.monthly_rent}.reverse
```

Comment:
Another way to solve this would be by using 
```ruby
apartments.sort {|apartment1, apartment2| apartment2.monthly_rent <=> apartment1.monthly_rent}
```
which would eliminate the need to call 2 methods on @apartments. It would sort it all at once in descending order rather than sorting it in ascending order and then reversing it. @wvicioso